Description: Remove "throw" specifications
 C++ throw specifications were deprecated in C++11.
 This patch removes them from the library.
Bug-Debian: https://bugs.debian.org/984342
Origin: upstream, https://git.savannah.gnu.org/cgit/src-highlite.git/commit/?id=904949c9026cb772dc93fbe0947a252ef47127f4
Author: Tom Tromey <tom@tromey.com>
Last-Update: 2020-06-10

--- a/lib/srchilite/fileutil.cc
+++ b/lib/srchilite/fileutil.cc
@@ -48,7 +48,7 @@
 // FIXME avoid using a global variable
 std::string start_path;
 
-string readFile(const string &fileName) throw (IOException) {
+string readFile(const string &fileName) {
     ifstream file(fileName.c_str());
 
     if (!file.is_open()) {
--- a/lib/srchilite/fileutil.h
+++ b/lib/srchilite/fileutil.h
@@ -27,7 +27,7 @@
  * @return the contents of the file
  * @throw IOException
  */
-string readFile(const string &fileName) throw (IOException);
+string readFile(const string &fileName);
 
 //char *read_file(const string &fileName);
 
